src/plugins/foxPage/components/Governance.tsx

```diff
@@ -9,6 +9,7 @@ import { bnOrZero } from 'lib/bignumber/bignumber'
 import { useGetGovernanceData } from '../hooks/getGovernanceData'
 
 const BOARDROOM_APP_BASE_URL = getConfig().REACT_APP_BOARDROOM_APP_BASE_URL
```

Comment:
remove boardroom now? @willyogo this is gone right?

Reply:
as discussed killin boardroom

Reply:
fuck it too much scope creep, remove this and replace with snapshot.js or metaforo api in a future PR